src/codegen/generateRouteFileInfoMap.spec.ts

```diff
@@ -108,4 +108,65 @@ describe('generateRouteFileInfoMap', () => {
         }"
       `)
   })
+
+  it('does not contain routes without components', () => {
```

Comment:
@posva This test already passed before my change in `generateRouteFileInfoMap.ts`